types/robust-segment-intersect/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for robust-segment-intersect 1.0
+// Project: https://github.com/mikolalysenko/robust-segment-intersect
+// Definitions by: masx200 <https://github.com/masx200>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+type Coordinate = [number, number];
+declare function a (
+        a0: Coordinate,
+        a1: Coordinate,
+        b0: Coordinate,
+        b1: Coordinate
+    ) :boolean;
+export = a;
```

Comment:
Current CI errors are formatting-related and should be fixed after running through Prettier:

```suggestion
type Coordinate = [number, number];
declare function a(
    a0: Coordinate,
    a1: Coordinate,
    b0: Coordinate,
    b1: Coordinate,
): boolean;
export = a;
```

Reply:
thank you